src/python/WMCore/ReqMgr/CherryPyThreads/StatusChangeTasks.py

```diff
@@ -11,36 +11,9 @@
 from WMCore.Services.WMStatsServer.WMStatsServer import WMStatsServer
 
 
-# FIXME: remove this function once MS Transferor is solid and in production
```

Comment:
This function was specific to the new statuses that are supposed to be managed by the MicroServices.
Yes, it was fairly tested ;) More tests needed in testbed though

Reply:
Ok, My understanding was that  there have been some period of time the both methods were actually running simultaneously and the new one have been complementary to the other.  So since you say that MCTransferor can handle the things and it will be the default method for the future, I do not see a reason why this function should not be removed.

Reply:
Yes, and I've made it as a separate function to make it easier for a future removal. Thanks